src/style.css

```diff
@@ -0,0 +1,73 @@
+@import url("https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap");
+
+* {
+  box-sizing: border-box;
+}
+
+html {
+  font-family: "Roboto", sans-serif;
+  font-size: 12px;
+}
+
+body,
+ul {
+  margin: 0;
+}
+
+.header {
+  display: flex;
+  justify-content: space-between;
+  align-items: center;
+  height: 60px;
+  padding: 0 50px;
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.05);
+}
+
+.header__logo-img {
+  display: block;
+}
+
+.navigation__item {
+  margin-right: 20px;
+}
+
+.navigation__item:last-child {
+  margin-right: 0;
+}
+
+.navigation__list {
+  list-style-type: none;
+  display: flex;
+  padding: 0;
+}
+
+.navigation__link {
+  text-transform: uppercase;
+  text-decoration: none;
+  color: #000;
+  position: relative;
+  display: block;
+  cursor: pointer;
+  padding: 23px 0;
```

Comment:
23 its magic number
https://medium.com/javascript-in-plain-english/is-magic-number-always-a-magic-number-c3b340b3f6cd
center your link not add paddings, make it centered so height does not has any effect on it